templates/new/web_dodo.py.txt

```diff
@@ -42,8 +43,26 @@ def task_batch():
         'actions': ["docker run %s python3 -m {project_name} --mode batch" % IMAGE]
     }}
 
+def task_train_local():
+    """Run training mode locally"""
+    return {{
+        'actions': ["%s -m {project_name} --mode train" % sys.executable]
+    }}
+
+def task_batch_local():
```

Comment:
Could we please use command names similar to Gradle i.e. camel case? You can use the `basename` parameter to set the name of the task rather than change the function name. See: https://pydoit.org/tasks.html#task-name